docs/labs/cloud-ide.md

```diff
@@ -64,5 +64,232 @@ All of the services are available to all Cloud IDE learners, for all flavours of
 Moreover, as an author, within Author IDE, you can add Buttons to your lab's markdown instructions to help learners open these pages with a simple button click.
 
 ### Building and Deploying to Code Engine
+If you would like to deploy any of the above-mentioned Embeddable AI NLP service and have it be available for anyone to use, you can follow these steps in order to deploy it. The deployment will be to IBM Cloud’s Code Engine. IBM Cloud Code Engine is a fully managed, cloud-native service for running containerized workloads on IBM Cloud. It allows developers to deploy and run code in a secure, scalable and serverless environment, without having to worry about the underlying infrastructure.
 
-_Instructions Coming Soon...._
+<details>
+  <summary><b><u>Deploying to Skills Network Code Engine</u></b></summary>
+
+The following steps allow you to test deploy to a IBM's Code Engine envrionement which is managed by Skills Network. This deployment is relatively easier and is recommended to quickly test out that if everything is working just fine.<br/>
+**Note: This deployment is temporary and is deleted after a few days.**
+
+##### 1. Create Code Engine Project
+
+In the left hand navigation pannel, there is an option for the Skills Network Toolbox. Simply open that and that expand the *CLOUD* section and then click on *Code Engine*. Finally cick on Create Project.
+
+![Create Code Engine Project](/img/labs/code-engine-create-project.png "Create Code Engine Project")
+
+##### 2. Click on Code Engine CLI Button
+
+From the same page simply click on Code Engine CLI button. This will open a new terminal and will login to a code engine project with everything alraedy set up for you.
+
+![Code Engine CLI](/img/labs/code-engine-cli-button.png "Code Engine CLI")
+
+##### 3. Choose and deploy your desired Embeddable AI image
+
+To get started, simply visit the [model catalog](https://www.ibm.com/docs/en/watson-libraries?topic=models-catalog) and choose the Watson's AI model you would like to use.
```

Comment:
Why are we pointing to official docs rather then the Embeddable AI section within Cloud IDE that lists the provided images (and models they provide).

Reply:
also since we host the images they won't be the same as what the official docs say (different ICR registry server and namespace).

They can also run: `ibmcloud cr image-list --restrict sn-labsassets` to see all of the provided images.

Also realize that the Embeddable AI section does _not_ list the images to use -- it should, we should likely add that in.